lib/ansible/modules/cloud/amazon/efs.py

```diff
@@ -565,7 +607,10 @@ def main():
     """
     argument_spec = ec2_argument_spec()
     argument_spec.update(dict(
+        encrypt=dict(required=False, type="bool", default=False),
         state=dict(required=False, type='str', choices=["present", "absent"], default="present"),
+        kms_key_id=dict(required=False, type='str', default=None),
+        purge_tags=dict(default=True, type='bool'),
```

Comment:
I'd like to pull @willthames and @ryansb into this conversation because I keep noticing purge_tags with different default values. I don't have a preference for whether this should default to true or false but since there's been some backlash over it I think there should be a documented decision so we can get some uniformity. See https://github.com/ansible/ansible/pull/31427#discussion_r147088175.
a) default to False and break backwards compatibility but err on the side of caution
b) default to True and keep compatibility but let users accidentally remove their tags

Reply:
I think defaulting to `false` is the safest option, and would be consistent with the purge_tags defaults in other modules.

Reply:
I think the majority do default to false. There are four that definitely do not - ec2_vpc_subnet, elb_target_group, elb_application_lb, and ec2_group all default purge_tags to true. Will those remain an anomaly or can we change the default?

Reply:
Linking another PR depending on this decision: https://github.com/ansible/ansible/pull/28513#discussion_r139060330

Reply:
I see - I do prefer having a `true` default, but had it in my head that the default was `false` most places. Since this module doesn't have backwards compatibility requirements, I'm ok with defaulting to `true` as in the guidelines.